src/nvim/globals.h

```diff
@@ -8,11 +8,10 @@
 // actually defined and initialized.
 #ifndef EXTERN
 # define EXTERN extern
-# define INIT(x)
+# define INIT(...)
 #else
 # ifndef INIT
```

Comment:
It checks for the empty `INIT` already defined. If I remove it, I get a bunch of missing declarations in the unit tests.